src/components/SpecialResolution/SpecialResolutionForm.vue

```diff
@@ -0,0 +1,463 @@
+                        ref="signingPersonFamilyNameRef"
+                        filled
+                        class="item"
+                        label="Last Name"
+                        id="person__last-name"
+                        v-model="signingPerson.familyName"
+                        :rules="lastNameRules"
+                        />
+                    </div>
+                  </v-form>
+                </v-col>
+              </v-row>
+
+              <v-row no-gutters>
+                <v-col cols="12" sm="3" class="pr-4">
+                  <label class="resolution-signature-vcard-title">Date Signed</label>
+                </v-col>
+                <v-col cols="12" sm="9" class="pt-4 pt-sm-0">
+                  <DatePickerShared
+                    ref="signatureDatePickerRef"
+                    title="Date Signed"
+                    :nudgeRight="40"
+                    :nudgeTop="85"
+                    :minDate="signatureDateMinStr"
+                    :maxDate="signatureDateMaxStr"
+                    :inputRules="signatureDateRules"
+                    @emitDateSync="onSigningDateSync"
+                  />
+                </v-col>
+              </v-row>
+            </v-card>
+
+        </section>
+      </v-card>
+    </v-card>
+  </v-card>
+</template>
+
+<script lang="ts">
+// this is a placceholder copied from AlterationSummary, Will add component when working on this page
+import { Component, Mixins, Watch } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { ActionBindingIF, HelpSectionIF, ResourceIF,
+  FormIF, PersonIF, EmptyPerson, CreateResolutionIF } from '@/interfaces/'
+import { DateMixin, SharedMixin, FilingTemplateMixin, CommonMixin } from '@/mixins/'
+import { HelpSection } from '@/components/common/'
+import { DatePicker as DatePickerShared } from '@bcrs-shared-components/date-picker/'
+
+@Component({
+  components: {
+    HelpSection,
+    DatePickerShared
+  }
+})
+export default class SpecialResolutionForm extends Mixins(
+  DateMixin
+
+) {
+  @Getter getResource!: ResourceIF
+  @Getter getBusinessFoundingDate!: Date
+
+  @Action setResolution!: ActionBindingIF
+  @Getter getCreateResolutionStep!: CreateResolutionIF
+  @Getter getComponentValidate!: boolean
+
+  // Refs
+  $refs!: {
+    resolutionDatePickerRef: DatePickerShared,
+    signatureDatePickerRef: DatePickerShared,
+    signingPersonFormRef: FormIF,
+  }
+
+  // Date properties
+  protected resolutionDateText: string = ''
+  protected signatureDateText: string = ''
+
+  protected MAX_RESOLUTION_TEXT_LENGTH: number = 1000
+  protected resolutionText: string = ''
+
+  protected signingPerson: PersonIF = null
+
+  get helpSection (): HelpSectionIF {
+    return this.getResource.changeData?.specialSpecialResolution?.helpSection
+  }
+
+  get getCreateResolutionResource (): any {
+    return this.getResource.changeData?.specialSpecialResolution?.sampleFormSection || null
```

Comment:
How about falling back to `{}` so that you can still de-reference it in the template (and remove the check if falsy)?